server/vulnerabilities/sync.go

```diff
@@ -24,23 +24,34 @@ import (
 	"github.com/go-kit/kit/log/level"
 )
 
+type SyncOptions struct {
+	VulnPath           string
+	CPEDBURL           string
+	CPETranslationsURL string
+	CVEFeedPrefixURL   string
```

Comment:
Let's document the fields. e.g if `CVEFeedPrefixURL` is empty, then `Sync` will use the default (nvd.nist.gov), etc.